snippets/prime_number.md

```diff
@@ -0,0 +1,24 @@
+---
+title: prime_number
+tags: prime_number,intermediate
+---
+
+Explain briefly what the snippet does.
+
+- Checks numbers from 2 to (given number - 1), If number is divisible by given numbers.

```

Comment:
The explanation could use some more work. I suppose the algorithm is similar to the [JS implementation](https://www.30secondsofcode.org/js/s/is-prime), so you could write something similar?